processFlows.dscp

```diff
@@ -34,12 +43,25 @@ token IssuedCert {
 pub fn initiate_cert || => |output: InitiatedCert| where {
   output.hydrogen_owner == sender,
   output.energy_owner != sender,
+  output.regulator != sender,
```

Comment:
we also probably want to assert that `regulator != energy_owner`

Reply:
this check also has been added.

Reply:
```
output.hydrogen_owner == sender,
output.energy_owner != sender,
output.regulator != sender,
```

Are the only checks on `initiate_cert` I'm seeing in github. I see there in `issue_cert` but they should be here as well

Reply:
ok so this doesn't work in the current compiler. Ignore for now